tests/composite_model_test.py

```diff
@@ -539,3 +540,61 @@ def test_failure_criteria_evaluation_default_unit_system(dpf_server):
     cfc = CombinedFailureCriterion("max stress", failure_criteria=[MaxStressCriterion()])
 
     failure_container = composite_model.evaluate_failure_criteria(cfc)
+
+
+def test_composite_model_with_imported_solid_model_assembly(dpf_server):
+    """
+    Imported solid models are slightly different if compared with shell parts
+    and standard solid models. For instance, they have analysis plies which are
+    not linked to a layered elements. These are called filler plies.
+
+    In addition, the show on reference surface feature used the skin of the
+    solid mesh instead of a predefined reference surface.
+    This general test ensures that the composite model can handle these.
+
+    The model has one shell part, one standard solid model and
+    an imported solid model which are all
```

Comment:
Something is missing here. Maybe:
```suggestion
    an imported solid model which are only consists of filler plies.
```